src/styles/blocks/card__review-stars.scss

```diff
@@ -0,0 +1,18 @@
+.card {
+  &__review-stars {
+    display: flex;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url('../images/star-active.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:last-child {
+      background-image: url('../images/star.svg');
+    }
```

Comment:
It's a good practice to use variables for colors, fonts, and static paths. Since you have a `variables.scss` file, consider defining your image paths there and use variables instead of direct paths.